web/src/helpers/helpers.ts

```diff
@@ -0,0 +1,64 @@
+import { helpers } from '@vuelidate/validators'
+
+/**
+ * Tiktok Link Regex
+ * https://stackoverflow.com/questions/74077377/regular-expression-to-match-any-tiktok-video-id-and-url
+ * Cases:
+ *   - https://m.tiktok.com/h5/share/usr/<id>.html
+ *   - https://m.tiktok.com/v/<id>.html
+ *   - https://www.tiktok.com/embed/<id>
+ *   - https://www.tiktok.com/share/user/<id>
+ *   - https://www.tiktok.com/trending?shareId=<id>
+ *   - https://www.tiktok.com/@user/video/<id>?is_copy_url=1&is_from_webapp=v1
+ *   - https://www.tiktok.com/@user/video/is_copy_url=1&is_from_webapp=v1&item_id=<id>
+ *   - https://vm.tiktok.com/<id>/
+ */
+export const tikTokLinkRegex = helpers.regex(
+    // eslint-disable-next-line no-useless-escape
+    /\bhttps?:\/\/(?:m|www|vm)\.tiktok\.com\/(?:.*\b(?:(?:usr|v|embed|user|video)\/|\?shareId=|\&item_id=)(\d+)\b|([\w\d]+))/gm
+)
+  
+/**
+ * Twitter Status Regex
+ * Cases:
+ *   - https://twitter.com/usr/status/<id>?s=24&t=DWBlw1gZk3FFWVQYOHUGYw
+ *   - https://mobile.twitter.com/usr/status/<id>
+ *   - https://twitter.com/usr/status/<id>
+ */
+export const twitterStatusRegex = helpers.regex(
+  /^https?:\/\/(?:mobile.)?twitter\.com\/(?:#!\/)?(\w+)\/status(?:es)?\/(\d+)(?:\/.*)?(\?(.+))?$/gm
+)
+
+/**
+ * Converts a given number to seconds or minutes
+ * 
+ * @param {string} value - A string representation of a number
+ * @return {string} The input number converted to either seconds or minutes, depending on its value
+ */
+export const convertToTime = (value: string): string => {
+  // Parse the input value as a number
+  const num = parseInt(value)
+
+  // Check if the number is greater than or equal to 60
+  if (num >= 60) {
```

Comment:
It would better if we could assign the number into a constant. avoid magic number. Can refer https://codehs.com/glossary/term/14